src/python/bcc/table.py

```diff
@@ -458,6 +457,34 @@ def items_delete_batch(self, keys=None):
             for _ in self.items_lookup_and_delete_batch():
                 return
 
+    def items_update_batch(self, keys: ct.Array, values: ct.Array):
+        """Update all the key-value pairs in the map provided.
+        The lists must be the same length, between 1 and the maximum number of entries.
+        """
+        # two ct.Array are expected
+        if not isinstance(keys, ct.Array) or not isinstance(values, ct.Array):
+            raise TypeError
+
+        batch_size = len(keys)
+
+        # check that batch between limits and coherent with the provided values
+        if (batch_size < 1 and batch_size > self.max_entries) or (batch_size != len(values)):
```

Comment:
LGTM. Only one comment: here 'and' should be 'or', right? Otherwise, for any `max_entries >= 1`, `batch_size < 1 and batch_size > self.max_entries` is always false.

Reply:
You're absolutely right, thanks for noticing! I fixed it also in items_delete_batch()